codechain/keystore/Pbkdf2Async.py

```diff
@@ -0,0 +1,8 @@
+import asyncio
+import hashlib
+
+
+async def pbkdf2_async(
```

Comment:
Why it is `async`? I understand that `hashlib.pbkdf2_hmac` is a CPU intensive blocking function, so it needed to be handled properly when it is needed. But making it just `async` doesn't make it be automatically off-loaded.
See: https://docs.python.org/3/library/asyncio-dev.html#running-blocking-code

Reply:
Okay, Done.

Reply:
I think we should just implement sync version only.